### ISSUE TITLE
Adguard + Adguard VPN do not work in integration mode

### ISSUE THREAD

**Issue by Assinhronizator** [AdguardTeam/AdGuardVPNForAndroid#296]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard VPN version

Adguard VPN v2.3 (100)

### Operating mode

Integration with AdGuard

### Environment

```markdown
- OS:Android 13 and Android 10
- Device:Samsung Galaxy S20 Ultra 5G and Huawei P40 Pro
- Firmware:CHC_13_0012 and C432E3R5P5
```

### Root access

- [ ] Yes, I have it.

### Issue Details

Steps to reproduce:
1. Install Adguard and Adguard VPN in integration mode
2. Enable Adguard VPN and choose any location
3. Go to the blocked site.
4. Site not opening

### Expected Behavior

When using Adguard and Adguard VPN in integration mode, all blocked sites in our country should open.

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

### Additional Information

Today I updated Adguard v4.0 for Android and Adguard VPN v2.3 (100) on all my phones (two Samsung Galaxy S20 Ultra 5G and one Huawei P40 Pro). When using Adguard VPN in compatibility mode, blocked sites are no longer opened, but the IP address is changed to the one selected in the program. Everything worked fine in the previous version. If you switch from compatibility mode to VPN mode in the advanced settings of Adguard VPN, everything works fine, blocked sites open.

I am attaching the log file below.
Problem reproduced at 11:28-11:29
[adguard_logs_4.0.912_1406_113014.zip](https://github.com/AdguardTeam/AdGuardVPNForAndroid/files/11743486/adguard_logs_4.0.912_1406_113014.zip)

**Comment by Versty** [comment]
@Assinhronizator This issue should be fixed in the AdGuard for Android 4.0 nightly 7. Would you mind checking on your device?

**Comment by Assinhronizator** [comment]
Hi,
I Installed Adguard 4.1 Nightly 7. The problem was gone. Now in 
compatibility mode, the Adgurd and Adgua VPN works good! I checked this 
on all my smartphones. All blocked sites are now opening.👍
Regards,
Artem


------ Исходное сообщение ------
От "Diana" ***@***.***>
Кому "AdguardTeam/AdGuardVPNForAndroid" 
***@***.***>
Копия "Assinhronizator" ***@***.***>; "Mention" 
***@***.***>
Дата 22.06.2023 20:29:39
Тема Re: [AdguardTeam/AdGuardVPNForAndroid] Adguard + Adguard VPN do not 
work in integration mode (Issue #296)
